packages/kernel-5.10/kernel-5.10.spec

```diff
@@ -72,6 +83,18 @@ Summary: Modules for the Linux kernel
 %description modules
 %{summary}.
 
+%if "%{_cross_arch}" == "x86_64"
+%package modules-neuron
+Summary: Modules for the Linux kernel with Neuron hardware
+Requires: %{name}
+Requires: %{_cross_os}ghostdog
+Requires: %{_cross_os}variant-platform(aws)
+Conflicts: %{_cross_os}variant-flavor(nvidia)
```

Comment:
Is there an actual conflict with these two or is this just using the `Conflicts` to get the right experience where `-nvidia` variants won't include this driver but the rest will?

Reply:
> Is there an actual conflict with these two or is this just using the Conflicts to get the right experience where -nvidia variants won't include this driver but the rest will?

I'm using the `Conflicts` here to indicate a license level conflict; the intent is to enforce that images only end up with one or the other. It's not required to have the right thing happen for the downstream variants in `bottlerocket-os/bottlerocket`.